release/create-package.sh

```diff
@@ -80,8 +80,8 @@ package_binary() {
 
   ${PROJECT_DIR}/build_distribution.sh
 
-  BIN_ORIGIN_NAME="rss-${RELEASE_VERSION}-hadoop2.8.tgz"
-  BIN_DIR_NAME="rss-${RELEASE_VERSION}-hadoop2.8"
+  BIN_ORIGIN_NAME="apache-uniffle-${RELEASE_VERSION}-incubating-rss-hadoop2.8.tgz"
```

Comment:
Maybe we don't need rss keyword.

Reply:
Thanks for the prompt reply and comments. 😄